official/mnist/mnist_tpu.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""MNIST model training using TPUs.
+
+This program demonstrates training of the convolutional neural network model
+defined in mnist.py on Google Cloud TPUs (https://cloud.google.com/tpu/).
+
+If you are not interested in TPUs, you should ignore this file.
```

Comment:
Perhaps add instructions of how to run this file? Judging from the [instructions of the wide deep model](https://cloud.google.com/ml-engine/docs/samples), it might be too lengthy though.

Reply:
(oops, correct link here: https://cloud.google.com/ml-engine/docs/getting-started-training-prediction)

Reply:
Will update when CloudTPUs are out of Alpha and are more accessible.